packages/components/src/context-tag/context-tag.tsx

```diff
@@ -1,20 +1,23 @@
+import { cloneElement } from 'react'
+
 import { ChevronRightIcon, PlayIcon } from '@status-im/icons'
 import { Stack, styled } from '@tamagui/core'
 import { View } from 'react-native'
 
 import { Avatar } from '../avatar'
 import { Text } from '../text'
 
+import type { AvatarProps } from '../avatar'
 import type { TextProps } from '../text'
+import type { IconProps } from '@status-im/icons'
 
 type Props = {
-  children?: React.ReactNode
-  size?: 24 | 32
+  size?: 20 | 24 | 32
   blur?: boolean
   outline?: boolean
 } & (
-  | { type: 'default'; user: { name: string; src: string } }
-  | { type: 'account'; user: { name: string; emoji: string } }
+  | { type: 'user'; user: { name: string; src: string } }
```

Comment:
This diverges from Design systems props.

Reply:
Will leave a comment in Figma too.

Reply:
Added comment to https://www.figma.com/file/WDrUK5Vf7BXV2XW6CdV5YB?node-id=973:17171&mode=design#541808836 and would say it all depends on https://github.com/status-im/status-web/pull/468#discussion_r1311266689.

Reply:
Yeah, I am not opposed to renaming to user overall, but it needs to be same for both Figma and React.

That seems like an issue of Avatar component. You'd say it's a user, but if you diverge from DS and new developer comes in they won't know what type they should use. It would made DX a mess.

Reply:
Reverted for now in order not to introduce unnecessary blocker. Will update accordingly based on the answers I'll get from design for all my questions. Thanks 👍.

Reply:
Type "Default" changed to "User" in designs too https://www.figma.com/file/IBmFKgGL1B4GzqD8LQTw6n/Design-System-for-Desktop%2FWeb?type=design&node-id=771-27404&mode=design&t=ISzoyEOMjUVv3Puk-4.